apps/cli-daemon/src/app/workspace/workspace.service.ts

```diff
@@ -157,4 +163,24 @@ export class WorkspaceService {
         return new InternalServerErrorException(err);
     }
   }
+
+  getWorkspaceConfiguration(): Partial<JSONSchema7> {
+    const configuration = configurationsPaths.reduce(
+      (state: JsonObject, currentPath) => {
+        const schemaValues = extractValueObject(
+          angularSchema,
+          currentPath.path,
+          currentPath.overridePath
+        );
+        return { ...state, ...schemaValues };
+      },
+      {} as JsonObject
+    );
+
+    return new SchemaDto({
+      title: 'Angular CLI Workspace Configuration',
+      description: 'Browser target options',
+      properties: { ...(configuration as JSONSchema7['properties']) },
```

Comment:
so that means the initial value of the reduce should also be JSONSchema7['properties'] and not JsonObject, no?

Reply:
im extracting properties data from the angular.json schema
formy needs to have data width `properties` to init it's schema.

Reply:
I see but why do you need to convert the type? just set the right type in the reduce function initial value

Reply:
the type is JSONSchema7 and the configuration is JsonObject.
https://github.com/sagold/json-schema-library#getchildschemaselection
i will use 
 const jsonSchema = new Draft07(angularSchema);

    const {root, prefix, sourceRoot} = jsonSchema.getSchema()['definitions']['project']['properties']
    const properties = new Draft07({root, prefix, sourceRoot}).getSchema()
    
    and we will have our types right.
    the API response with JSONSchema7 (that formly is relaying on when converting to formly format)